synapse/tests/test_lib_grammar.py

```diff
@@ -14,6 +14,7 @@
 # flake8: noqa: E501
 
 Queries = [
+    'emit $foo stop',
```

Comment:
This is also valid
```
emit$foo stop
```

Reply:
Is this a suggestion for changing it to require a space or just making an observation? ( the purpose of these tests isn't to cover every syntax variant possible but to make sure that future changes dont break the AST generated by any specific expression :D )